php/ad-servers/class-ad-layers-dfp.php

```diff
@@ -493,7 +493,7 @@ private function ad_unit_js( $ad_layer ) {
 					$custom_targeting = null;
 					if ( ! empty( $this->ad_units[ $unit_key ] ) ) {
 						$custom_targeting = $this->ad_units[ $unit_key ];
-					} elseif ( empty( $this->ad_units[ $unit_key ] ) && ! empty( $ad_unit['custom_targeting'] ) ) {
+					} elseif ( ! empty( $ad_unit['custom_targeting'] ) ) {
```

Comment:
It's actually not a change in logic, it was just something I cleaned up while I was reading the method. The flow was:

```php
if ( ! empty( $this->ad_units[ $unit_key ] ) ) {
} elseif ( empty( $this->ad_units[ $unit_key ] ) && ! empty( $ad_unit['custom_targeting'] ) ) {
}
```

The first `empty()` check in the elseif is redundant to the `if` clause.